Source/Scene/Camera.js

```diff
@@ -2984,8 +2984,11 @@ Camera.prototype.getPickRay = function (windowPosition, result) {
     result = new Ray();
   }
 
+  const scene = this._scene;
   const frustum = this.frustum;
-  if (
+  if (defined(scene.canvas.style) && scene.canvas.style["display"] === "none") {
+    return undefined;
```

Comment:
The jsDoc comment for this function will need to be modified to include that `undefined` will be returned if the pickRay cannot be determined.

Reply:
Nice catch! I want to make sure that we keep our documentation consistent. I just added a commit that should address this issue.